package review202;

import java.util.Scanner;

public class Review202 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
//		catch exception
//		try{
//			myExceptionTest();
//		}catch(Exception ex){
//			ex.printStackTrace(System.out);
//		}
		
		Scanner input = new Scanner(System.in);
//		System.out.println("Enter two integers ");
//		int x = input.nextInt();
//		int y = input.nextInt();
		int x = 5, y = 0;
		
		try{
//			myExceptionTest();
			System.out.println(x + "/" + y + "=" + division(x, y));
			myExceptionTest();
		}catch(Exception ex){
//			ex.printStackTrace(System.out);
//			ex.getStackTrace();
			System.out.println(ex.toString());
			
		}

	}
	
	
//	declare exception
	public static void myExceptionTest() throws Exception{
//		throw exception
		throw new MyTestException("This is a testing");
		
	}
	
	public static double division(int x, int y) throws Exception{
		if (y == 0){
			throw new ArithmeticException("Divide by zero...");
		}
		return x/y;
	}

	
}
